### PR TITLE
feat: add serverless structure and automate deploy

### PR DESCRIPTION
**Pull request by trigolos**
1. Task: https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md
2. Screenshot:
image
)
3. Links:
   - S3 Backet - http://manual-aws-bucket.s3-website-us-east-1.amazonaws.com (403 - Access Denied)
   - Manual CloudFront - https://d38crrzlp4wftp.cloudfront.net
   - Deployed by script: https://d390yilayuoqgi.cloudfront.net

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+service: my-rs-service
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
```
Comment (vvkotov) [inline serverless.yml:23]: `region` could be set to `eu`, cause default is `us`

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+resources:
+  Resources:
+    ## Specifying the S3 Bucket
+    WebAppS3Bucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ${self:custom.s3BucketName}
+        AccessControl: PublicRead
+        WebsiteConfiguration:
+          IndexDocument: index.html
```
Comment (vvkotov) [inline serverless.yml]: If you defined a variable `indexDocument`, then I guess it could be used here. And for errors as well

**Review by vvkotov — APPROVED** [review]
Would be great to add `.serverless` to .gitignore file

File: serverless.yml (hunk 1 above)
Comment (trigolos) [reply serverless.yml:23]: region added, thanks

File: serverless.yml (hunk 2 above)
Comment (trigolos) [reply serverless.yml]: You are right, fixed. Thanks

**Pull request merged**